File: SoloLearn Codes/CMYK to RGB/CmykToRgb.java
import java.util.*;
import java.util.stream.*;

public class CmykToRgb {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double[] CMYK = Arrays.stream(new double[4])
                .map(x -> sc.nextDouble())
                .toArray();
        System.out.println(IntStream
                .range(0, 3)
                .mapToDouble(a -> 255 * (1 - CMYK[a]) * (1 - CMYK[3]))
                .mapToObj(a -> String.valueOf((int)Math.round(a)))
                .collect(Collectors.joining(",")));
    }
}

File: SoloLearn Codes/CMYK to RGB/cmyk_to_rgb.c
#include <stdio.h>

int main() {
    float CMYK[4];
    for (int i = 0; i < 4; i++) {
        scanf("%f", &CMYK[i]);
    }
    for (int i = 0; i < 3; i++) {
        printf((i == 0) ? "%.f" : ",%.f", 255 * (1 - CMYK[i]) * (1 - CMYK[3]));
    }
 
    return 0;
}